CODEOWNERS

```
@@ -0,0 +1,10 @@
+# See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file
+#
+*                   @equinix/governor-metal-client-interfaces
+/cmd/migration-tool @equinix/governor-metal-client-interfaces
+*metal*             @t0mk @equinix/governor-metal-client-interfaces
+*fabric*            @equinix/governor-digin-fabric
+*ecx*               @equinix/governor-digin-fabric
+*connection_e2e*    @equinix/governor-digin-fabric
+*equinix_network*   @equinix/network-edge-team
+**/edge-networking  @equinix/network-edge-team
```

The NE team was migrated to Governor after this PR was opened; this is the new team for them.

```suggestion
*equinix_network*   @equinix/governor-ne-network-edge-engineering
**/edge-networking  @equinix/governor-ne-network-edge-engineering
```